litex/soc/interconnect/wishbone.py

```diff
@@ -35,7 +35,7 @@
 
 
 class Interface(Record):
-    def __init__(self, data_width=32, adr_width=30):
+    def __init__(self, data_width=32, adr_width=31):
```

Comment:
@gsomlo: i think this is breaking tests and CI, could we keep 30 as default for now?

Reply:
I don't think we can have a memory region larger than 2GB (`0x8000_0000`) (e.g., [here](https://github.com/enjoy-digital/litex/blob/master/litex/soc/integration/soc.py#L848)) without increasing the `addr_width` by one bit, to `31`.

We can hold off on applying this PR until you have a chance to
  1. decide whether bumping to `31` is, in fact, OK to do
  2. update CI to stop complaining about it :)

But I think the change is necessary to cover the entire 4GB address space with a single memory region.
**Unless** we can somehow cover a single slave port with *two* memory regions (each of them 2GB in size)? Not sure if that's possible, but I'm certainly of the opinion that bumping `adr_width` would be cleaner...